streams/src/test/java/org/apache/kafka/streams/integration/StreamsUncaughtExceptionHandlerIntegrationTest.java

```diff
@@ -198,35 +195,6 @@ public void shouldShutdownSingleThreadApplication() throws Exception {
         testShutdownApplication(1);
     }
 
-    @Test
-    public void shouldShutDownClientIfGlobalStreamThreadWantsToReplaceThread() throws Exception {
```

Comment:
Why are we removing this test? I think we should just update the code to call to new/remaining overload of `addGlobalStore()` instead.